public/css/menu.css

```diff
@@ -25,8 +25,28 @@ h1 {
   margin: 0;
 }
 
-.category img {
+.image-wrapper {
   width: 100%;
   height: 15rem;
+  position: relative;
+}
+
+.category img {
+  width: 100%;
+  height: 100%;
   object-fit: cover;
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  transition: transform 0.2s;
+  visibility: visible;
+  opacity: 1;
+  transition: opacity 0.3s;
+}
+
+.hidden {
+  visibility: hidden !important;
```

Comment:
I figured out why this wasn't working... In `global.css` we have a different `.hidden` which is `display: none !important`.  And something that is display none will never load, and therefore never blurup.